src/tests/electron/tests/results-view.test.ts

```diff
@@ -178,7 +230,13 @@ describe('ResultsView', () => {
     });
 
     it('export report button exists', async () => {
+        await openResultsView();
         await setupWindowForCommandBarReflowTest('wide');
         await resultsView.waitForSelector(ResultsViewSelectors.exportReportButton);
     });
+
+    async function openResultsView(): Promise<void> {
```

Comment:
Potential refactor (maybe a separate PR): If this returned `Promise<ResultsViewController>`, then resultsView could be defined inside each test case.